### PR TITLE
Use github actions to run unit tests

### PR DESCRIPTION
**Pull request by teadur**
example workflow

### CODE REVIEW

**Comment by teadur** [comment]
Dropped testing on ubuntu 20 ( problems with wkhtmltopdf-bin ) and ruby 3.0 (tests fail with core dump)
Some work to support ruby 3.0, dropped mina/listener gems to speed up deploy.
Please Squash and merge after review.

**Comment by yulgolem** [comment]
Got a question - is it possible to point CodeClimate test coverage checks to one of GA test envs?

**Review by yulgolem — APPROVED** [review]

**Comment by teadur** [comment]
> Got a question - is it possible to point CodeClimate test coverage checks to one of GA test envs?
> 
> BTW checked speed of GA deploy - it's about 3 to 5 minutes compared to current 40+ minutes in Travis. Nice result, thank you.
GA now runs CodeClimate checks, not sure why the total-coverage falls -13.5% thou :|

**Comment by karlerikounapuu** [comment]
> > Got a question - is it possible to point CodeClimate test coverage checks to one of GA test envs?
> > BTW checked speed of GA deploy - it's about 3 to 5 minutes compared to current 40+ minutes in Travis. Nice result, thank you.
> 
> GA now runs CodeClimate checks, not sure why the total-coverage falls -13.5% thou :|
> https://codeclimate.com/github/internetee/registry/compare/331ade988c2e891d8addc634e105bf68318defc3...9bcba24b50766726edb74ebb689792f4572562f1#ratings

`bundle exec rake test test/*` doesn't run all the tests for some reason. `bundle exec rails test test/* `should run everything that's included :)

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        PG_PASSWORD: password
+        PG_PORT: ${{ job.services.postgres.ports[5432] }}  
+        RAILS_ENV: test
+        COVERAGE: true
+        DISABLE_SPRING: 1
+      run: |
+        cp config/application.yml.sample config/application.yml
+        cp config/database_travis.yml config/database.yml
+        bundle exec rake db:setup:all
+        bundle exec rake test test/*
```
Comment (karlerikounapuu) [inline .github/workflows/ruby.yml:72]: `bundle exec rake test test/*` doesn't run all the tests for some reason. `bundle exec rails test test/* `should run everything that's included :)

**Review by karlerikounapuu — APPROVED** [review]

**Comment by yulgolem** [comment]
Changed test command to up the coverage, rebased from master & squashed, removed TravisCI support.
PR is ready to merge.

**Comment by teadur** [comment]
> Changed test command to up the coverage, rebased from master & squashed, removed TravisCI support.
> PR is ready to merge.

Nice, thank you :)

**Pull request merged**